blog.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Blog</title>
+    <link rel="stylesheet" href="./styles/master.css">
+    <style>
+      body {
+         background-color: lightblue;
+      }
+    </style>
+  </head>
+
+  <body>
+    <header>
+      <img src="images/adoptadog-logo.svg" width="200" alt="adoptadog logo">
```

Comment:
Don't use any styling attributes in your HTML it should all be in your CSS file.